src/js/OTSession.coffee

```diff
@@ -121,7 +121,7 @@ class TBSession
     element = streamElements[ elementId ]
     if(element)
       element.parentNode.removeChild(element)
```

Comment:
This line removes the `subscriber` div completely and causes an error when another endpoint publishes and this endpoint tries to subscribe. Please use the helper function `this.removeElement(element)` instead of `element.parentNode.removeChild(element)`

Reply:
@msach22 Updated and used the helper method